pkg/autodetect/main_test.go

```diff
@@ -6,7 +6,7 @@ import (
 	"testing"
 	"time"
 
-	"github.com/googleapis/gnostic/OpenAPIv2"
+	openapi_v2 "github.com/googleapis/gnostic/OpenAPIv2"
```

Comment:
nit: I think we don't use underscores in import names

Reply:
I think this came from the `make fmt`, which puts the package name from the source file instead of the path.

https://github.com/googleapis/gnostic/blob/master/OpenAPIv2/OpenAPIv2.go#L17